frontend/src/components/SilvicultureSearch/Openings/SearchScreenDataTable/index.tsx

```diff
@@ -105,36 +111,38 @@ const SearchScreenDataTable: React.FC<ISearchScreenDataTable> = ({
   }, [rows, totalItems]);
   
   // Function to handle row selection changes
-  const handleRowSelectionChanged = (openingId: string) => {
+  const handleRowSelectionChanged = (rowId: string) => {
     setSelectedRows((prevSelectedRows) => {
-      if (prevSelectedRows.includes(openingId)) {
+      if (prevSelectedRows.includes(rowId)) {
         // If the row is already selected, remove it from the selected rows
-        return prevSelectedRows.filter((id) => id !== openingId);
+        return prevSelectedRows.filter((id) => id !== rowId);
       } else {
         // If the row is not selected, add it to the selected rows
-        return [...prevSelectedRows, openingId];
+        return [...prevSelectedRows, rowId];
       }
     });
   };
 
-  const { displayNotification } =  useNotification();
+  const handleRowClick = (openingId: string) => {
+    // Call the mutation to mark as viewed
+    markAsViewedOpening(openingId, {
+      onSuccess: () => {
+        // setToastText(`Successfully marked opening ${openingId} as viewed.`);
+      },
+      onError: (err: any) => {
+        // setToastText(`Failed to mark as viewed: ${err.message}`);
```

Comment:
Are we missing something here? It seems like this should trigger a toast notification